ala/act_make_configs.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env ruby
+require 'optimist' #for options parsing
+
+opts = Optimist::options do
+    opt :devices, "REQUIRED: List of devices to test. Ex: /dev/nvme3n1p1,/dev/nvme1n1p4", :default => ""
+    opt :directory, "Directory to save config files to. (No trailing /, directory must exist)", :default => "/opt"
+    opt :x, "How many 'X' configs you want to generate. At the default of 120, it will generate 120 actconfigs.", :default => 120
```

Comment:
You mentioned in the [readme](https://github.com/aerospike/act/pull/35/files?utf8=%E2%9C%93&diff=unified#diff-072e92c2457c780c24651dab0c26032dR9) that you intend to only generate 12 configs by default.